pelops/features/hog.py

```diff
@@ -20,14 +21,38 @@ def produce_features(self, chip):
         img = self.get_image(chip)
         img = img.resize(self.image_size, Image.BICUBIC)
         img_x, img_y = img.size
+
+        # Calculate histogram of each channel
```

Comment:
I think that's what I'm doing. I now initialize an array of `-1`s and fill it with features from up to 3 channels.